content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -203,6 +203,16 @@ Follow these steps to verify the deployment:
 
 Follow these steps to access the Kubeflow central dashboard:
 
+1. You need the
```

Comment:
Suggested revision:

Use the following command to grant yourself the [IAP-secured Web App User](https://cloud.google.com/iap/docs/managing-access) role:

Reply:
I couldn't commend on line 172:

```diff
- To deploy kubeflow just run
+ To deploy Kubeflow, run the following command:

[triple back tick]
make apply
[triple back tick ]
``` 

I would add also colons (":") where you see them missing before each command, but it's probably not a priority here.